contrib/core-contract-tests/tests/bns/name_register.test.ts

```diff
@@ -323,54 +323,55 @@ describe("name revealing workflow", () => {
     expect(result).toBeErr(Cl.int(2022));
   });
 
-  it("should successfully register", () => {
-    const name = "bob";
-    const merged = new TextEncoder().encode(`${name}.${cases[0].namespace}${cases[0].salt}`);
-    const sha256 = createHash("sha256").update(merged).digest();
-    const ripemd160 = createHash("ripemd160").update(sha256).digest();
-    simnet.callPublicFn("bns", "name-preorder", [Cl.buffer(ripemd160), Cl.uint(2560000)], bob);
-
-    const register = simnet.callPublicFn(
-      "bns",
-      "name-register",
-      [
-        Cl.bufferFromUtf8(cases[0].namespace),
-        Cl.bufferFromUtf8(name),
-        Cl.bufferFromUtf8(cases[0].salt),
-        Cl.bufferFromUtf8(cases[0].zonefile),
-      ],
-      bob
-    );
-    expect(register.result).toBeOk(Cl.bool(true));
-
-    const resolvePrincipal = simnet.callReadOnlyFn(
-      "bns",
-      "resolve-principal",
-      [Cl.standardPrincipal(bob)],
-      alice
-    );
-    expect(resolvePrincipal.result).toBeOk(
-      Cl.tuple({
-        name: Cl.bufferFromUtf8("bob"),
-        namespace: Cl.bufferFromUtf8("blockstack"),
-      })
-    );
-
-    const nameResolve = simnet.callReadOnlyFn(
-      "bns",
-      "name-resolve",
-      [Cl.bufferFromUtf8(cases[0].namespace), Cl.bufferFromUtf8(name)],
-      alice
-    );
-    expect(nameResolve.result).toBeOk(
-      Cl.tuple({
-        owner: Cl.standardPrincipal(bob),
-        ["zonefile-hash"]: Cl.bufferFromUtf8(cases[0].zonefile),
-        ["lease-ending-at"]: Cl.some(Cl.uint(16)),
-        ["lease-started-at"]: Cl.uint(6),
-      })
-    );
-  });
+  // temp disabled, focusing on importing clarunit correctly
+  // it("should successfully register", () => {
```

Comment:
I believe you can use `skip` instead of commenting it out:
```ts
it.skip("should successfully register", () => {
```

What is still needed to be able to run these two skipped tests?

Reply:
I forgot to look at the state of the TS files when I took over the PR. I will check them now.